library/src/main/java/com/telefonica/mistica/compose/card/postercard/PosterCardTopActions.kt

```diff
@@ -0,0 +1,79 @@
+package com.telefonica.mistica.compose.card.postercard
+
+import androidx.compose.foundation.Image
+import androidx.compose.foundation.background
+import androidx.compose.foundation.clickable
+import androidx.compose.foundation.layout.Arrangement
+import androidx.compose.foundation.layout.Box
+import androidx.compose.foundation.layout.Row
+import androidx.compose.foundation.layout.Spacer
+import androidx.compose.foundation.layout.fillMaxWidth
+import androidx.compose.foundation.layout.padding
+import androidx.compose.foundation.layout.size
+import androidx.compose.foundation.layout.width
+import androidx.compose.foundation.layout.wrapContentSize
+import androidx.compose.foundation.shape.CircleShape
+import androidx.compose.runtime.Composable
+import androidx.compose.ui.Alignment
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.draw.clip
+import androidx.compose.ui.graphics.Color
+import androidx.compose.ui.graphics.ColorFilter
+import androidx.compose.ui.layout.ContentScale
+import androidx.compose.ui.platform.testTag
+import androidx.compose.ui.res.painterResource
+import androidx.compose.ui.unit.dp
+
+@Composable
+internal fun PosterCardTopActions(modifier: Modifier = Modifier, firstTopAction: TopActionData?, secondTopAction: TopActionData?) {
+    Row(
+        modifier = modifier
+            .fillMaxWidth()
+            .padding(top = 16.dp, start = 16.dp, end = 16.dp, bottom = 8.dp),
+        horizontalArrangement = Arrangement.End,
+        verticalAlignment = Alignment.CenterVertically
+    ) {
+        firstTopAction?.let {
+            TopAction(topActionData = it)
+        }
+        secondTopAction?.let {
+            Spacer(modifier = Modifier.width(16.dp))
+            TopAction(topActionData = it)
+        }
+    }
+}
+
+@Composable
+internal fun TopAction(topActionData: TopActionData) {
+    with(topActionData) {
+        Box(
+            modifier = Modifier
+                .testTag(topActionData.testTag.orEmpty())
```

Comment:
Since you are using `with` scope function you can use:
```suggestion
                .testTag(testTag.orEmpty())
```

Reply:
Applied here --> https://github.com/Telefonica/mistica-android/pull/398/commits/91ca8546f3096629afe6e6c50d3c82ae7a6b7377